pkg/cri/server/service_linux.go

```diff
@@ -87,6 +88,14 @@ func (c *criService) initPlatform() (err error) {
 		}
 	}
 
+	if c.config.EnableCDI {
```

Comment:
nitpick: If the CRI plugin is disabled can it happen that this gets omitted altogether and then later non-CRI/fat clients cause trouble by using `WithCDI()` externally, which is then not running with the desired configuration in effect ?

Reply:
@klihub Thank you for the review! I've moved WithCDI to the pkg/cri/opts as suggested by @fuweid . It should resolve this issue.